knowledge_repo/app/models.py

```diff
@@ -497,9 +507,11 @@ def update_metadata_from_kp(self, kp):
 
         self.status = kp.status
 
+        self.private = 0
+        # we do this check so that no header (None) and False are treated the same
         if headers.get('private', ''):
```

Comment:
This one should be: `headers.get('private', False)`, since the types are translated to boolean if present. Obviously doesn't change logic, so not a biggie.